docs/book/src/development/releasing.md

```diff
@@ -0,0 +1,37 @@
+# Releasing Cluster API Provider for CloudStack
+
+## Prerequisites:
+
+1. Please install the following tools :
+    - [go][go]
+    - [Docker][docker-install]
+    - [gcloud][gcloud-install]
+
+2. Set up and log in to gcloud by running `gcloud init`
```

Comment:
how does one log in?

Reply:
It's initiated automatically as part of the init

Reply:
Are there any specific gcloud permissions we need to publish the CAPC repo? Or can anyone do it?